inline_Ob.cc

```diff
@@ -296,7 +297,35 @@ namespace Chroma
                 	nF[nu][mu] = F[nu][mu];
                 	nF[mu][nu] = -nF[nu][mu];
            		}	
-       		 }    
+       		 }   
+
+/*
+                        for(int i=1;i<4; i++)
+                        {
+                                O1 += F[3][i]*un*Fn[z][i]*adj(un);
+                                O10 += F[3][i]*F[z][i];
+                        }
+ 
+                                multi1d<int> Coord_loc, Coord_shift ;
+                                Coord_loc.resize(Nd);
+				Coord_shift.resize(Nd);
+                                Coord_loc[3] = t;
+                                Coord_shift[3] = t;
+                                for(int x = 0; x < Layout::lattSize()[0];x++)
+                                  for(int y = 0; y < Layout::lattSize()[1];y++)
+                                    for(int Z = 0; Z < Layout::lattSize()[2];Z++)
+                                        {
+                                                Coord_loc[0] = x;
+                                                Coord_loc[1] = y;
+                                                Coord_loc[2] = Z;
+						Coord_shift[0] = x;
+                                                Coord_shift[1] = y;
+                                                Coord_shift[2] = Z;
+						Coord_shift[z] = (Coord_shift[z]+1)%Layout::lattSize()[z];
+						 =peekSite(F[3][i], Coord_loc)*peekSite(u[z],Coord_loc)*peekSite(Fn[z][i], Coord_shift)*peekSite(adj(u[z]),Coord_loc);	
+					}
+
+*/
```

Comment:
Trying to calculate operators with the known link and F_\mu\nu information at all the sites without using shift function,